### PR TITLE
[ACA-1822] input search options

### PR DESCRIPTION
**Pull request by suzanadirla**
## PR Checklist
Please check if your PR fulfills the following requirements:

```
- [x] The commit message follows our guidelines: https://github.com/Alfresco/alfresco-content-app/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
```

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Application / Infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: https://issues.alfresco.com/jira/browse/ACA-1822

## What is the new behavior?
new Finding Libraries (the Search Input - part)

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/app/components/search/search-input-control/search-input-control.component.scss [hunk 1]
```diff
@@ -1,8 +1,59 @@
-.adf-clear-search-icon-wrapper {
-  width: 1em;
+@mixin aca-search-input-control-theme($theme) {
```
Comment (pionnegru) [inline src/app/components/search/search-input-control/search-input-control.component.scss]: please consider separating theme from layout

File: src/app/app.routes.ts [hunk 2]
```diff
@@ -208,6 +208,27 @@ export const APP_ROUTES: Routes = [
...
+            component: SearchResultsComponent,
+            data: {
+              title: 'APP.BROWSE.SEARCH.TITLE',
+              reuse: true
+            }
+          },
+          {
+            path: 'preview/:nodeId',
+            loadChildren:
+              'src/app/components/preview/preview.module#PreviewModule',
```
Comment (DenysVuika) [inline src/app/app.routes.ts]: Please make it relative to the app.module like other routes. Otherwise it breaks for certain scenarios.

Comment (DenysVuika) [inline src/app/app.routes.ts]:
```suggestion
              './components/preview/preview.module#PreviewModule',
```

File: src/app/components/search/search-input-control/search-input-control.component.scss [hunk 3]
```diff
@@ -1,8 +1,50 @@
...
+      font-size: 24px;
+      padding-right: 0;
+    }
+  }
+}
+
+.app-prefix-search-icon-wrapper {
+  height: 6px;
+  margin: 14px 13px;
+  float: left;
```
Comment (pionnegru) [inline src/app/components/search/search-input-control/search-input-control.component.scss]: why do we need a float? :)

Comment (suzanadirla) [reply src/app/components/search/search-input-control/search-input-control.component.scss]: I had trouble styling that item (might be related to [this issue](https://github.com/angular/material2/issues/10093) - even though it's closed) and float is the thing that fixed it... :)

**Pull request merged**